CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Unreleased
 
+IMPROVEMENTS:
+
+* Add server.extraConfig and client.extraConfig values as hashes on Server
+  StatefulSet and Client Daemonset annotations respectively. This recreates
+  the server/client pod when the server/client extraConfig is updated via `helm upgrade` [[GH-550](https://github.com/hashicorp/consul-helm/pull/550)]
+
+BREAKING CHANGESL
```

Comment:
```suggestion
BREAKING CHANGES:
```

Reply:
This section should be above improvements (https://github.com/hashicorp/engineering-docs/blob/master/consul/changelog-management.md#format) so that folks don't miss it.